modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -289,8 +289,12 @@ definition {
 				value1 = "${name}");
 		}
 
-		if (isSet(parameterOption)) {
-			ObjectRelationships.editRelationshipsParameter(parameterOption = "${parameterOption}");
+		if (isSet(parameter)) {
```

Comment:
When you changed this variable you forgot to pass it on the test case. Can you change this: `parameterOption = "Self Relationship",` `for this: parameter = "Self Relationship",`